dapps/digital-twin-detail/src/components/container/List.ts

```diff
@@ -104,7 +119,7 @@ export default class ContainerListComponent extends mixins(EvanComponent) {
       : `${fileList.files.length} ${this.$t('_twin-detail.data.files')}`;
   }
 
-  getValues(): any[] {
+  get values(): any[] {
```

Comment:
Already tried this during implementation, but got problems with the refreshing on dispatcher finish. Can you check, if the list gets correctly refreshed, when the dispatcher has finished?